gapis/gfxapi/vulkan/vulkan.api

```diff
@@ -4607,6 +4607,11 @@ cmd VkResult vkResetDescriptorPool(
     VkDevice                   device,
     VkDescriptorPool           descriptorPool,
     VkDescriptorPoolResetFlags flags) {
+  sets := DescriptorPools[descriptorPool].DescriptorSets
```

Comment:
Can you make a note of the behaviors you're relying on?
If I ever get around to finishing the LLVM compiler, I'm going to have to be careful to either correct this code or keep the same behaviors.